src-tauri/src/lib.rs

```diff
@@ -40,6 +42,21 @@ pub fn check_mod_version_number(path_to_mod_folder: String) -> Result<String, an
     Ok(mod_version_number.to_string())
 }
 
+// I intend to add more linux related stuff to check here, so making a func
+// for now tho it only checks `ldd --version`
+// - salmon
+#[cfg(target_os = "linux")]
+pub fn linux_checks_librs() -> bool {
+    let mut linux_compatabile: bool = true; // a variable that starts true and will be set to false if any of the checks arent met
+
+    // check `ldd --version` to see if glibc is up to date for northstar proton
+    let lddv = linux::check_glibc_v();
+    if lddv < 2.33 { linux_compatabile = false };
```

Comment:
Pretty sure this needs a `#[cfg(target_os = "linux")]` or otherwise it won't compile on Windows ^^